web/src/test/java/com/microsoft/applicationinsights/web/extensibility/modules/WebSessionTrackingTelemetryModuleTests.java

```diff
@@ -121,24 +114,6 @@ public void testNoSessionCreatedWhenValidSessionExists() throws Exception {
         Assert.assertNull(cookiesContainer.getSessionCookie());
     }
 
-    @Test
-    public void testNewSessionIsCreatedWhenCookieSessionExpired() throws Exception {
-        sessionCookieFormatted = HttpHelper.getFormattedSessionCookieHeader(true);
-
-        CookiesContainer cookiesContainer = HttpHelper.sendRequestAndGetResponseCookie(sessionCookieFormatted);
-        SessionCookie sessionCookie = cookiesContainer.getSessionCookie();
-
-        Assert.assertNotNull(sessionCookie);
-        Assert.assertFalse(sessionCookieFormatted.contains(sessionCookie.getSessionId()));
-    }
-
-    @Test
-    public void testNewSessionIsCreatedWhenCookieCorrupted() throws Exception {
-        CookiesContainer cookiesContainer = HttpHelper.sendRequestAndGetResponseCookie("corrupted;session;cookie");
-
-        Assert.assertNotNull("Session cookie shouldn't be null.", cookiesContainer.getSessionCookie());
-    }
-
     @Ignore
     @Test
     public void testWhenSessionExpiredSessionStateEndTracked() throws Exception {
```

Comment:
We should talk to Harel/Yossi to understand what should be done with the SessionState now, since we're not generating anymore.